src/Microsoft.ML.Data/MLContext.cs

```diff
@@ -79,6 +79,13 @@ public sealed class MLContext : ISeededEnvironment
         /// </summary>
         public ComponentCatalog ComponentCatalog => _env.ComponentCatalog;
 
+        private static string _tempFilePath = System.IO.Path.GetTempPath();
+        public static string TempFilePath
+        {
+            get { return _tempFilePath; }
+            set { _tempFilePath = value; }
+        }
```

Comment:
nit: you can write the whole things here as:
```C#
public static string TempFilePath {get; set; } = System.IO.Path.GetTempPath();
```

Reply:
That is what I did originally, but ML.NET has a "style cop" and it complains if I do that. I can look into disabling it though for that line.